test/julia/test_pyjulia.py

```diff
@@ -52,3 +57,80 @@ def test_pyjulia_pipeline():
     # check with analytical value
     p_opt = obj.get("p_opt")
     assert np.allclose(result.optimize_result[0].x, p_opt)  # noqa: S101
+
+
+def test_petabJL_interface():
+    """Test the interface to PEtab.jl with provided solutions from julia."""
+    model_name = "boehm_JProteomeRes2014"
+    yaml_file = f"doc/example/{model_name}/{model_name}.yaml"
```

Comment:
Please use an absolute path based on `__file__` so pytest can be invoked from any directory. Also below.